src/labels/CustomLabels.labels

```diff
@@ -4968,6 +4992,136 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Error completing the specified API call</shortDescription>
         <value>Error completing the {0} api: {1}</value>
     </labels>
+    <labels>
+        <fullName>batchNumberSettingsDescActivation</fullName>
```

Comment:
@kenlewis-sfdo - this block of labels should be exactly what you have in your current Batch Auto Number branch! It's all copy-paste so with luck we can merge either one first and the related diff in the other will just disappear